### DISCUSSION TITLE
Add HTML5 tags and attributes

### DISCUSSION THREAD

**Discussion by beargiles** [jhy/jsoup#2226] [Ideas]
Add HTML5 tags and attributes.

HTML5 adds many elements that provide semantic context for parsers - e.g., <article>, <aside>, <summary> etc at the structural level, or <mark>, <time>, etc. at the text level. It also adds media controls (<audio>, <video>, etc.) and graphics (<svg>, etc.) Many of these elements include `src` or `icon` elements.

I have a POC implementation.

**Comment by jhy** [comment]
#2284
